bin/HFO

```diff
@@ -138,6 +140,8 @@ def parseArgs():
                  help='Server provides full-state information to agents.')
   p.add_argument('--seed', dest='seed', type=int, default=-1,
                  help='Seed the server\'s RNG. Default: time.')
+  p.add_argument('--messageSize', dest='messageSize', type=int, default=1000,
```

Comment:
Please change this to '--message-size' in order to conform with style of the other arguments.